src/sdk/Audio/AudioConfig.ts

```diff
@@ -105,6 +106,28 @@ export abstract class AudioConfig {
         throw new Error("Not Supported Type");
     }
 
+    /**
+     * Creates an AudioConfig object from a standard Readable stream.
+     * Useful when streaming audio from external network sources (e.g., ffmpeg UDP stream into a Docker container).
+     * @member AudioConfig.fromReadableStream
+     * @function
+     * @public
+     * @param {Readable} audioStream - Custom audio input from a Readable stream. Currently, only WAV / PCM is supported.
+     * @param {AudioStreamFormat} format - The audio data format in which audio
+     *        will be returned from the stream's read() method.
+     * @returns {AudioConfig} The audio output configuration being created.
+     */
+    public static fromReadableStream(audioStream: Readable, format?: AudioStreamFormat): AudioConfig {
+        const pushStream = AudioInputStream.createPushStream(format);
+
+        audioStream.on('readable', () => {
+            const buffer = audioStream.read();
+            pushStream.write(buffer);
+        });
+
```

Comment:
Nothing ever closes the push stream. That can lead to recognition hanging when the Readable runs out of data.

Reply:
Excellent point! I've updated the code to close the push stream when `read()` returns `null`.

Are there any other cases you believe should be handled? (e.g., do you have thoughts around `.on('error', () => { ??? })`)